nethserver-directory.spec

```diff
@@ -44,6 +44,8 @@ rm -rf %{buildroot}
 %ghost %attr(644,root,root) /etc/pam.d/password-auth-nh
 %ghost %attr(644,root,root) /etc/pam.d/system-auth-nh
 %config %attr (0440,root,root) %{_sysconfdir}/sudoers.d/20_nethserver_directory
+%attr (0750,root,root) /usr/libexec/nethserver/slapd-create-dhparam
```

Comment:
Our RPM macro should set 0755 perms and the binary is outside of PATH settings. I think there's no practical need to restrict the executable permissions for normal users.

```suggestion
```